pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
I just wanted to say that I have not lost this thread, I'm just buried in
the midst of a few crazy weeks.

On Mon, Feb 23, 2015 at 8:31 PM, Brian Grant <notifications@github.com>
wrote:

> In pkg/api/types.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4480#discussion_r25228080>
> :
>
> > @@ -705,6 +713,8 @@ type ServiceSpec struct {
> >  	// not be changed by updates.
> >  	PortalIP string `json:"portalIP,omitempty"`
> >
> > +	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
>
> Looking through the code, it looks like rewrite is only valid when
> createExternalLoadBalancer is true and PublicIPs is empty. Therefore, it
> appears it belongs in ServiceStatus. Errors for invalid uses would be
> useful, also.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4480/files#r25228080>
> .
>